src/views/ConfigEditor/FormParts/DefaultValues/DefaultValues.tsx

```diff
@@ -14,6 +14,10 @@ jsonData, onSwitchToggle, onFieldChange, externalProps
 
   useEffect(() => {
     const doRequest = async () => {
+      if (!jsonData.dataSourceUrl.startsWith('http://') && !jsonData.dataSourceUrl.startsWith('https://')) {
```

Comment:
this is wrong fix IMHO, change approach

I requested check options only after press "Save & Test"

for example, if you press "use CORS" or "use POST"
and don't press "Save & Test" nothing shall happen

do you will make request for every press button in any control?
password for example?
username?

Reply:
@lunaticusgreen 
wrong behavior still not fixed
![image](https://github.com/user-attachments/assets/13406e92-8978-4224-b0ab-206439dc1a77)

look detailed to 
https://github.com/Altinity/clickhouse-grafana/issues/587